src/python.rs

```diff
@@ -498,6 +498,21 @@ impl<'p> Python<'p> {
     pub fn xdecref<T: IntoPyPointer>(self, ptr: T) {
         unsafe { ffi::Py_XDECREF(ptr.into_ptr()) };
     }
+
+    /// Lets the Python interpreter check for pending signals and invoke the
+    /// corresponding signal handlers. This can run arbitrary Python code.
+    ///
+    /// If an exception is raised by the signal handler, or the default signal
+    /// handler raises an exception (such as `KeyboardInterrupt` for `SIGINT`),
+    /// an `Err` is returned.
```

Comment:
Can this doc maybe also mention that it's a safe wrapper about `PyErr_CheckSignals`? I think it can be helpful for some people to know this without having to look at the source.

Reply:
Sure!